### PR TITLE
added add tags feature

### PR DESCRIPTION
**Pull request by vinayak-trivedi**
- added a feature to add category and level to the user, which creates generic items tags for those tasks

### CODE REVIEW

File: src/task/index.html [hunk 1]
```diff
@@ -106,7 +106,7 @@
             <option value="ASSIGNED">Assigned</option>
           </select>
         </div>
-        <div class="inputBox">
+        <div class="inputBox hidden">
```
Comment (vinayak-trivedi) [inline src/task/index.html:109]: hiding this until we have the api

**Review by heyrandhir — APPROVED** [review]
LGTM 🙂👍🏻

File: src/task/index.html (hunk 1 above)
Comment (alishvagadiya) [reply src/task/index.html:109]: Add a comment in the file, instead of in PR,
No one goes back and check PR review comments.

Comment (ankushdharkar) [reply src/task/index.html:109]: @alishvagadiya I think the comment is meant for the reviewers of just this PR. It's probably there as an optional information.

**Review by ankushdharkar — APPROVED** [review]
Thank you for the changes.

**Pull request merged**